### PR TITLE
GT-2201 Add image that links to homepage within tools

### PR DESCRIPTION
**Pull request by caleballdrin**
## Description
We are wanting to add an image to the top center of the header within tools as shown in the [Figma](https://www.figma.com/file/B0iiHKuUtWkPoNIXSn63BLd5/GT-knowgod.com-V1?type=design&node-id=0%3A1&mode=design&t=8mrneRffdJHce1W0-1) that users can click to bring them back to the home page.
https://jira.cru.org/secure/RapidBoard.jspa?rapidView=150&view=detail&selectedIssue=GT-2201

## Changes I Made
- Add logo and link
- Change CSS to reformat for small screens

### CODE REVIEW

File: src/app/new-page/page.component.html [hunk 1]
```diff
@@ -4,14 +4,17 @@
   <div class="outerbox">
     <header id="toolHeader">
       <div class="header tc relative">
+        <app-sharing-modal [book]="selectedBookName"></app-sharing-modal>
+        <span class="logo">
```
Comment (canac) [inline src/app/new-page/page.component.html:8]: You may be able to remove this `<span>` wrapper and put the class directly on the `<a>`.

**Review by canac — APPROVED** [review]
Great work!

File: src/app/new-page/page.component.html (hunk 1 above)
Comment (caleballdrin) [reply src/app/new-page/page.component.html:8]: ahh yep that worked

**Review by dr-bizz — CHANGES_REQUESTED** [review]
This looks good, just one thing, on responsive, can you make the logo shrink a little, so it doesn't force the header onto two lines?
I know they have mobile apps, but it's good practice to ensure websites work on mobile.
Screenshot 2023-11-13 at 8 34 24 AM

**Comment by wrandall22** [comment]
Sorting languages looks to be unrelated to this PR.

**Comment by dr-bizz** [comment]
> @dr-bizz I intentionally made the header go to 2 lines when it is on mobile. That is the same thing they did on the home page.. 

Gotcha!

**Review by dr-bizz — APPROVED** [review]

**Review by canac — APPROVED** [review]

**Comment by dr-bizz** [comment]
@caleballdrin I think I have reviewed everything. Unless I'm not seeing a change you've made

**Pull request merged**